samples/bring-your-own-container/README.md

```diff
@@ -0,0 +1,32 @@
+# Bring Your Own Container Sample
+
+This sample demonstrates the ability for customers to start training jobs using their own training scripts, packaged in a SageMaker-compatible container, using the Amazon SageMaker Operator for Kubernetes. 
+
+## Prerequisites
+
+This sample assumes that you have already configured an EKS cluster with the operator. It also assumes that you have installed `kubectl` - you can find the links on our [installation page](https://sagemaker.readthedocs.io/en/stable/amazon_sagemaker_operators_for_kubernetes.html#prerequisites).
+
+In order to follow this script, you must first create a training script packaged in a Dockerfile that is [compatible with Amazon SageMaker](https://docs.aws.amazon.com/sagemaker/latest/dg/amazon-sagemaker-containers.html). The [Distributed Mask R-CNN sample](https://github.com/awslabs/amazon-sagemaker-examples/tree/master/advanced_functionality/distributed_tensorflow_mask_rcnn), published by the SageMaker team, contains a predefined training script and helper bash scripts for reference.
+
+## Preparing the Training Script
+
+### Uploading your Script
+
+All SageMaker training jobs must be packaged in a container with all necessary dependencies pre-installed and with the training scripts referencing the acceptable input and output paths. This container should be uploaded to a container repository accessible from within your AWS account (for example [ECR](https://aws.amazon.com/ecr/)). When uploaded correctly, you should have a URL and tag associated with the container image - this will be needed for the next step.
```

Comment:
* `(for example [ECR](https://aws.amazon.com/ecr/))`. AFAIK ECR is the only Docker repository currently supported in SageMaker, so should change language from "for example" style to a hard requirement.

* `All SageMaker training jobs must be packaged in a container with all necessary dependencies pre-installed and with the training scripts referencing the acceptable input and output paths.` Nit to be more precise in language, a SageMaker training job is not packaged in a container...
* `When uploaded correctly, you should have a URL and tag associated with the container image - this will be needed for the next step.` Might be helpful to give an example ECR URL and tag.